ReportPortal.ATM/TestProject/Features/DashboardAddWidget.feature

```diff
@@ -0,0 +1,15 @@
+﻿Feature: Dashboard Add Widget feature
+
+User is able to add widget to dashboard
+
+@Smoke
+Scenario: Add a Widget
+	Given I log in to ReportPortal
+	And I navigate to all dashboards page
+	When I click on add new dashboard
+	And I enter dashboard Test and Hola
+	And I click on add a widget
+	And I Add Launch statistics chart widget type and enter TestWidget name
+	And I click on add a widget
+	And I Add Overall statistics widget type and enter TestWidget2 name
```

Comment:
One observation here:

If the widget's name cannot be repeated, it would be beneficial to not specify the widget's name in the step but, rather, generate a random name within the step definition's code, so that the widget step is always new and unique.

This is only a recommendation, if there is no issue running this test case over and over, then there is no need to change anything.